tutorials/editor/default_key_mapping.rst

```diff
@@ -255,7 +255,7 @@ Text Editor
 +---------------------------+--------------------------+----------------------------+-------------------------------------------------+
 | Fold/Unfold Line          | :kbd:`Alt + F`           | :kbd:`Alt + F`             | ``script_text_editor/toggle_fold_line``         |
 +---------------------------+--------------------------+----------------------------+-------------------------------------------------+
-| Clone Down                | :kbd:`Ctrl + D`          | :kbd:`Cmd + Shift + C`     | ``script_text_editor/clone_down``               |
+| Clone Down                | :kbd:`Ctrl + Shift + D`  | :kbd:`Cmd + Shift + C`     | ``script_text_editor/clone_down``               |
```

Comment:
```suggestion
| Clone Down                | :kbd:`Ctrl + Shift + D`  | :kbd:`Cmd + Shift + C`     | ``script_text_editor/clone_down``               |
| Duplicate Selection       | :kbd:`Ctrl + Shift + D`  | :kbd:`Cmd + Shift + C`     | ``script_text_editor/duplicate_selection``      |
```

Reply:
So in that case should the clone down editor shortcut entry just be replaced with "Duplicate selection" and "clone down" removed since it doesn't exist?

Reply:
Sorry accidentally duplicated